var/spack/repos/builtin/packages/intel-parallelstudio/package.py

```diff
@@ -0,0 +1,112 @@
+from spack import *
+import sys, os, re
+
+from spack.pkg.builtin.intel import IntelInstaller, filter_pick
+
+class IntelParallelstudio(IntelInstaller):
```

Comment:
Maybe rename this to `IntelParallelStudio` and change the directory name to `intel-parallel-studio`